src/Components/Navigationlink/index.css

```diff
@@ -0,0 +1,11 @@
+.Navigationlink {
```

Comment:
In this case, to respect SUIT Convention with Pascal Case, you could use `NavigationLink`, instead of `Navigationlink`